### PR TITLE
HEATMAP

### PR DESCRIPTION
**Pull request by shivesh01**
```needreview```
#1531 
I have made changes suggested by @sampoder 
Thanks for your suggestion

### CODE REVIEW

File: workshops/HEATMAP/README.md [hunk 1]
```diff
@@ -0,0 +1,353 @@
+---
+name: 'Make your own Heat map'
+description: 'Data visualization with Sea born'
```
Comment (sampoder) [inline workshops/HEATMAP/README.md:3]: Hmm could we make this sound more professional?

File: workshops/HEATMAP/README.md [hunk 2]
```diff
@@ -0,0 +1,353 @@
+---
+name: 'Make your own Heat map'
```
Comment (sampoder) [inline workshops/HEATMAP/README.md:2]:
```suggestion
name: 'Make Your Own Heat Map'
```

standard capitalisation

File: workshops/HEATMAP/README.md [hunk 3]
```diff
@@ -0,0 +1,353 @@
...
+author: '@shivesh01'
+---
+
+# Heat Map
+
+![Slideshow](https://cloud-49dp118l9.vercel.app/0slidshow.gif)
+
+
+
+We use data everywhere. Your smartphone for instance, or the satellite navigation system in your car; maybe the car itself, what not?
```
Comment (sampoder) [inline workshops/HEATMAP/README.md:13]:
```suggestion
We use data everywhere. Your smartphone for instance, and the satellite navigation system in your car are a couple of examples.
```

File: workshops/HEATMAP/README.md [hunk 4]
```diff
@@ -0,0 +1,353 @@
...
+
+![Slideshow](https://cloud-49dp118l9.vercel.app/0slidshow.gif)
+
+
+
+We use data everywhere. Your smartphone for instance, or the satellite navigation system in your car; maybe the car itself, what not? 
+Data that we share among ourselves mostly is visual data which may be text, audio, video or graphs. Which is essential for us to continue discovering new things and keep ourselves connected. 
+Visual data such as graphs plays an important role in understanding complex data and even helps us to remember for a lit longer. Graphs can be of many kinds but we are here to learn about heat maps and even make them while learning.
+
+[Click here to see working HEATMAP DEMO on repl](https://repl.it/@ShiveshSingh/Heatmap#main.py)
```
Comment (sampoder) [inline workshops/HEATMAP/README.md:17]:
```suggestion
[Click here to see a working demo of today's project](https://repl.it/@ShiveshSingh/Heatmap#main.py)
```

File: workshops/HEATMAP/README.md [hunk 5]
```diff
@@ -0,0 +1,353 @@
...
+
+
+# Introduction 
+
+In this workshop, you will be using the python language and its libraries to create a stunning heatmap from scratch. Even if you don't know the python language no worries this tutorial will guide you and keep you interested. Most importantly you will learn about heatmap. let's talk about what is a heat map? and how to make one like data scientists do!🤗.
+The heat maps are graphical representation of data where values are represented by shades of color. Heat maps make it easy to visualize complex data and understand it at a glance:
+Even you can visualize the data, your data too!
+
+
+> One step at a time is all it takes to get you there.
```
Comment (sampoder) [inline workshops/HEATMAP/README.md:47]:
```suggestion
```

File: workshops/HEATMAP/README.md [hunk 6]
```diff
@@ -0,0 +1,353 @@
...
+We use data everywhere. Your smartphone for instance, or the satellite navigation system in your car; maybe the car itself, what not? 
+Data that we share among ourselves mostly is visual data which may be text, audio, video or graphs. Which is essential for us to continue discovering new things and keep ourselves connected. 
+Visual data such as graphs plays an important role in understanding complex data and even helps us to remember for a lit longer. Graphs can be of many kinds but we are here to learn about heat maps and even make them while learning.
+
+[Click here to see working HEATMAP DEMO on repl](https://repl.it/@ShiveshSingh/Heatmap#main.py)
+
+
+
+
+# Index
```
Comment (sampoder) [inline workshops/HEATMAP/README.md:22]: let's remove this, not much point imo

File: workshops/HEATMAP/README.md [hunk 7]
```diff
@@ -0,0 +1,353 @@
...
+
+* Resources for datasets
+
+* Hacking
+
+
+
+# Introduction 
+
+In this workshop, you will be using the python language and its libraries to create a stunning heatmap from scratch. Even if you don't know the python language no worries this tutorial will guide you and keep you interested. Most importantly you will learn about heatmap. let's talk about what is a heat map? and how to make one like data scientists do!🤗.
```
Comment (sampoder) [inline workshops/HEATMAP/README.md:42]: Capitalise Python

Comment (sampoder) [inline workshops/HEATMAP/README.md:42]: Punctuation at the end is also wrong

File: workshops/HEATMAP/README.md [hunk 8]
```diff
@@ -0,0 +1,353 @@
...
+The heat maps are graphical representation of data where values are represented by shades of color. Heat maps make it easy to visualize complex data and understand it at a glance:
```
Comment (sampoder) [inline workshops/HEATMAP/README.md:43]:
```suggestion
The heat maps are graphical representation of data where values are represented by shades of color. Heat maps make it easy to visualize complex data and understand it at a glance.
```

File: workshops/HEATMAP/README.md [hunk 9]
```diff
@@ -0,0 +1,353 @@
...
+Even you can visualize the data, your data too!
```
Comment (sampoder) [inline workshops/HEATMAP/README.md:44]:
```suggestion
```

File: workshops/HEATMAP/README.md [hunk 10]
```diff
@@ -0,0 +1,353 @@
...
+
+
+> One step at a time is all it takes to get you there.
+
+
+# Getting started
+
+You just need to set up an account on repl. you can run this program on your Python idle too. Idle like Pycharm, Atom, Jupyter notebook and Visual studio e.t.c. Some basic knowledge is required for the programming and you are ready to go. I have designed this workshop sweet and simple for you. It will just take 15-20 minutes to complete the workshop.
```
Comment (sampoder) [inline workshops/HEATMAP/README.md:52]: Capitals at the start of every sentence, also awkward phrasing here

File: workshops/HEATMAP/README.md [hunk 11]
```diff
@@ -0,0 +1,353 @@
...
+Ready, Get set & Go! 
+
+## Step zero!
+
+### Setting up an account on repl
```
Comment (sampoder) [inline workshops/HEATMAP/README.md:57]: Combine these two titles

File: workshops/HEATMAP/README.md [hunk 12]
```diff
@@ -0,0 +1,353 @@
...
+
+- Go to the repl website and sign up 
+[Repl website](https://repl.it/signup) and create a repl
```
Comment (sampoder) [inline workshops/HEATMAP/README.md:60]: make this flow better

File: workshops/HEATMAP/README.md [hunk 13]
```diff
@@ -0,0 +1,353 @@
...
+```
+**Output**
+![Scatter plot](https://cloud-ikhasazgz.vercel.app/3scatter_plot.png )
+Fantastic! You have created your first plot🥳
+
+
+
+
+# Step 2. Relplot
+
```
Comment (sampoder) [inline workshops/HEATMAP/README.md:135]: make this section flow better

File: workshops/HEATMAP/README.md [hunk 14]
```diff
@@ -0,0 +1,353 @@
...
+plt.show() 
+```
+Wow! You have created your relp plot 🥳👏🏻
+**Output**
+![Relplot](https://cloud-ikhasazgz.vercel.app/2relp_plot.png)
+
+
+# Step 3. Violin Plot
+
+*A Violin Plot is used to visualize the distribution of the data and its probability density.  Show the distribution shape of the data.*
```
Comment (sampoder) [inline workshops/HEATMAP/README.md:166]: awkward phrasing

File: workshops/HEATMAP/README.md [hunk 15]
```diff
@@ -0,0 +1,353 @@
...
+**Output**
+![Heatmap](https://cloud-ikhasazgz.vercel.app/0heatmap.png)
+
+### Marvelous! You have created your Heat map plot ###
+
+
+# voilà!
+
+**Congratulations!✌️ you have completed the workshop. Share it with your friends and family.
+Frankly share with everyone🤗✌️🥳👏🏅🌇🎊**
```
Comment (sampoder) [inline workshops/HEATMAP/README.md:292]: frankly is a weird word to use here

File: workshops/HEATMAP/README.md [hunk 16]
```diff
@@ -0,0 +1,353 @@
...
+grid_kws = {"height_ratios": (.9, .05), "hspace": .3}
+f, (ax, cbar_ax) = plt.subplots(2, gridspec_kw=grid_kws)
+ax = sns.heatmap(flights, ax=ax,
+cbar_ax=cbar_ax,
+cbar_kws={"orientation": "horizontal"})
+plt.show()
+```
+
+
+# Keep Hacking!
```
Comment (sampoder) [inline workshops/HEATMAP/README.md:340]: need three demos and code links

**Review by sampoder — CHANGES_REQUESTED** [review]
Hey! Thanks for submitting another version!

My main concern with this workshop is that it isn't flowing well, and sounds too tutorial and not like a project.

Alongside that the phrasing still needs work on and some grammar errors. I've done my best to point them out below.

Thanks!

**Comment by shivesh01** [comment]
#1535 
Hey @sampoder, I have made some changes. Hope you like it. 
Duplicate of #

**Pull request closed without merging**